NET.A.Carting/Domain/Entities/Cart.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Domain.Common;
+
+namespace Domain.Entities
+{
+    public class Cart : BaseAuditableEntity
```

Comment:
According to functional requirements Cart should have a list of Items.
Properties below need to be part of Item entity

`The following actions should be supported:
Get list of items of the cart object.
Add item to cart.
Remove item from the cart.
Each item contains the following info:
Id – required, id of the item in external system (see Task 2), int.
Name – required, plain text.
Image – optional, URL and alt text.
Price – required, money.
Quantity – quantity of items in the cart.`